content/docs/cml/start-gitlab.md

```diff
@@ -0,0 +1,117 @@
+# Using CML on GitLab
+
+Here, we'll walk through a tutorial to start using CML on GitLab. For
+simplicity, we'll show the demo in GitLab Actions, but these instructions are
+valid for all supported Continuous Integration (CI) systems (with exceptions as
+noted!).
+
+1. Fork our
+   [example project repository](https://gitlab.com/iterative.ai/example_cml).
+   Click on Fork and select what namespace to fork the project to.
+
+   ![](/img/gitlab_fork_cml_project.png)
+
+   > 💡 If just starting to use GitLab, you may receive a warning at the top of
+   > the page regarding SSH keys. You do not need to use SSH keys for the
+   > tutorial so you can click: "Don't show me again."  
+   > [Click this link for more information on SSH keys.](https://docs.gitlab.com/ee/ssh/)
```

Comment:
💅 this looks strange in the rendered page for some reason:

![image](https://user-images.githubusercontent.com/1477535/108442701-1bbe3380-721d-11eb-8d1e-df63562ef6b1.png)

Maybe make the link shorter so it fits in the 3rd line.

Reply:
But wait... I'm not sure we need this note at all 🙂 

Users should probably figure out their SSH key situation separately.